app/src/components/pages/Register.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useEffect, useState } from 'react';
+import Button from '@mui/material/Button';
+import TextField from '@mui/material/TextField';
+import Box from '@mui/material/Box';
+import { Grid, Typography } from '@mui/material';
+
+interface RegisterForm {
+  email: string;
+  username: string;
+  password: string;
+  confirmPassword: string;
+}
+
+function Register() {
+  const [hasErrors, setHasErrors] = useState<boolean>(false);
+  const [formData, setFormData] = useState<RegisterForm>({
+    email: '',
+    username: '',
+    password: '',
+    confirmPassword: ''
+  });
+
+  const createAccount = (e: { preventDefault: () => void }) => {
+    e.preventDefault();
+    console.log('Creating account: ', formData);
+  };
+
+  useEffect(() => {
+    (formData.password === formData.confirmPassword && formData.password.length > 0) || formData.email.length === 0
+      ? setHasErrors(false)
+      : setHasErrors(true);
+  }, [formData.password, formData.confirmPassword]);
+
+  return (
+    <Box
+      component="form"
+      sx={{
+        width: 300
+      }}
+      m="auto"
+      justifyContent="center"
+      autoComplete="off"
+    >
+      <Grid container spacing={2} marginTop={2}>
+        <Grid item>
+          <Typography variant="h4">Create an Account</Typography>
+        </Grid>
+        {hasErrors && (
+          <Grid item>
+            <Typography variant="body1" color="error">
+              Passwords do not match.
+            </Typography>
+          </Grid>
+        )}
+        <Grid item>
+          <TextField
+            id="outlined-required"
+            label="UVic Email"
+            placeholder="someone@uvic.ca"
+            style={{ width: 300 }}
+            value={formData.email}
+            onChange={(e) => setFormData({ ...formData, email: e.target.value })}
+          />
+        </Grid>
+        <Grid item>

```

Comment:
Looking at the API doc, we only need to collect a username and password. I would drop the V# entry and just switch the email placeholder to emphasize they're supposed to enter their uvic username (ie mine would be `bmazerolle`, my email is `bmazerolle@uvic.ca`.

Reply:
I think we can keep the uvic email section since it's more realistic to have a registration form requiring email.  I don't think we would have points docked for making the authentication process more realistic.

Reply:
If the requirements show we should be collecting a UVIC username rather than an email then I think it makes more sense to change the email field to a username field. I don't mind keeping the V number box for style points even though it's not wired up to anything.

Reply:
Looks good now!